### ISSUE TITLE
BlueLaTeX on Mozilla Science Lab's Collaborate

### ISSUE THREAD

**Issue by BillMills** [gnieh/bluelatex#215]
Hey all,

Thanks for submitting BlueLaTeX to Collaborate! Your [listing](http://www.mozillascience.org/projects/gnieh-bluelatex) is now live.

Have you heard about our [Global Sprint](http://www.mozillascience.org/global-sprint-2015) coming up June 4-5? Open source & open science projects around the world will be sprinting on their projects in this two-day hackathon, where we invite people to host sites worldwide to discover what people are working on and get involved. Let me know if you'd like to represent BlueLaTeX, we'd love to have you.

**Comment by satabin** [comment]
Hi @BillMills 

Thanks for  proposing, I wasn't aware of this program and think it would be very interesting to participate.
I will read more about it and let you know

@monperrus would you like to join the discussion?

**Comment by monperrus** [comment]
Hi Bill,

>  Open source & open science projects around the world will be sprinting on their projects in this two-day hackathon, where we invite people to host sites worldwide to discover what people are working on and get involved. Let me know if you'd like to represent BlueLaTeX, we'd love to have you

We'd love to have BlueLaTeX at the Mozilla Science Lab Global Sprint 2015!

--Martin

**Comment by BillMills** [comment]
Awesome - add your name to the list around line 75 [here](https://etherpad.mozilla.org/sciencelab-2015globalsprint); also, have a look at the site list on the same etherpad, to see if there are any host sites near you; if not, feel free to join remotely or if you can host a site, add it to the list at the site list. I'll flag your project as participating in the Sprint so it shows up on the [sprint project list](http://www.mozillascience.org/global-sprint-2015). Looking forward to it!

**Comment by BillMills** [comment]
*oh, I forgot - will one of you be available to answer questions, merge pull requests etc during those two days? We like to have a project lead on hand (remotely is fine) to help organize volunteers.

**Comment by satabin** [comment]
Hi Bill,

I will check this with my employer and let you know.

**Comment by satabin** [comment]
Hi Bill,

We will be able toe participate to the Sprint.
I will update the pad.

**Comment by BillMills** [comment]
Awesome! You're all queued up; check out the [map](http://www.mozillascience.org/global-sprint-2015) to see if there's an open site near you - if so, feel free to drop in, and if not, consider hosting your own, or participate remotely from wherever is convenient. For the date, try and have a few issues and projects ready to go for different people to jump in on; some smaller issues for beginners, and some more open ended ones for more experienced programmers. I'll be reaching out to the twitterverse over the next month to draw attention to all the Sprint projects; if there are promotional materials I should be linking (blog posts, videos, articles...), just let me know.

**Comment by satabin** [comment]
Is there a place (the pad maybe?) where to put the IRC channel we will be available on?

**Comment by BillMills** [comment]
Yep, include it in your project description on the pad - I'll be remixing that content into project listings for people to browse once submissions close.
